App.js

```diff
@@ -77,8 +114,17 @@ const App = () => {
       </View>
       <ScrollView>
         {todo?.map((item, i) => {
-          return <TodoItem item={item} key={i} handleCheck={handleCheck} />;
-        })}
+          return (
+            <TodoItem
+              item={item}
+              key={i}
```

Comment:
Why are you sending key as a prop
remove return and set it to single line return